web/src/beta/lib/reearth-ui/components/Button/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { FC, useCallback, useState } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+export type ButtonProps = {
+  appearance?: "primary" | "secondary" | "dangerous" | "simple";
+  disabled?: boolean;
+  size?: "normal" | "small";
+  iconButton?: boolean;
+  icon?: string; // TODO: Icon Name, Use Icon Component
+  title?: string;
+  onClick?: () => void;
+};
+
+export const Button: FC<ButtonProps> = ({
+  appearance = "secondary",
+  icon = "",
+  disabled = false,
+  size = "normal",
+  iconButton = false,
+  title = "",
+  onClick,
+}) => {
+  const [isMouseOver, setIsMouseOver] = useState<boolean>(false);
+  const [isActive, setIsActive] = useState<boolean>(false);
+
+  const handleButtonClick = useCallback(() => {
+    if (onClick) onClick();
+  }, [onClick]);
+
+  return (
+    <StyledButton
+      disabled={disabled}
+      appearance={appearance}
+      size={size}
+      hover={isMouseOver}
+      active={isActive}
+      iconButton={iconButton}
+      onMouseOver={() => setIsMouseOver(true)}
+      onMouseLeave={() => setIsMouseOver(false)}
+      onMouseDown={() => setIsActive(true)}
+      onMouseUp={() => setIsActive(false)}
+      onClick={handleButtonClick}>
+      {/* TODD: Use Icon Component based on icon */}
+      {icon}
+      {!iconButton && title}
+    </StyledButton>
+  );
+};
+
+const StyledButton = styled("button")<{
+  size: "normal" | "small";
+  appearance: "primary" | "secondary" | "dangerous" | "simple";
+  hover: boolean;
+  active: boolean;
+  disabled: boolean;
+  iconButton: boolean;
+}>(({ disabled, appearance, size, hover, active, iconButton, theme }) => ({
+  display: "flex",
+  flexDirection: "row",
+  alignItems: "center",
+  gap: `${theme.spacing.small}`,
+  border:
+    appearance === "simple"
+      ? "none"
+      : disabled
+      ? `1px solid ${theme.content.weaker}`
+      : `1px solid ${
+          active || hover
+            ? "transparent"
+            : appearance === "secondary"
+            ? theme.outline.weak
+            : appearance === "primary"
+            ? theme.primary.main
+            : theme.dangerous.main
+        }`,
+  padding:
+    size === "small"
+      ? iconButton
+        ? `${theme.spacing.micro}px`
+        : `${theme.spacing.smallest}px ${theme.spacing.small}px`
+      : iconButton
+      ? `${theme.spacing.small}px`
+      : `${theme.spacing.small}px ${theme.spacing.large}px`,
+  borderRadius: size === "small" ? `${theme.spacing.smallest}px` : "6px",
```

Comment:
use theme.radius instead of theme.spacing size === "small" ? `${theme.radius.small}px` : `${theme.radius.normal}px`